apps/src/jigsaw/blocks.js

```diff
@@ -172,13 +171,16 @@ exports.install = function (blockly, blockInstallOptions) {
   delete blockly.Blocks.procedures_ifreturn;
 };
 
-function generateBlankBlock(blockly, skin, name, hsv, width, hasAppend) {
+function generateBlankBlock(blockly, skin, name, hsv, width, hasAppend, label) {
   blockly.Blocks[name] = {
     helpUrl: '',
     init: function () {
       this.setHSV.apply(this, hsv);
       this.appendDummyInput()
-        .appendTitle(new blockly.FieldImage(skin.blank, width, 54));
+        .appendTitle(new blockly.FieldImage(skin.blank, (width-8)/2, 54))
```

Comment:
Why the change of `width` -> `(width-8)/2`?

Tiny nit: calculate this once above, and store it in a descriptive variable.

Reply:
Previously, `width` was used to supply the total width of the block, but this is only accounting half of it, but this function is only used here, so I can switch to just passing in a smaller value for width if that would be clearer

Reply:
Yeah, I think it's a little cleaner to update the places where we call `generateBlankBlock` above with the new value.

Reply:
Or just inline it as always 36 and remove the parameter, now that it's always the same value!